apps/meteor/ee/app/livechat-enterprise/server/api/outbound.ts

```diff
@@ -1,55 +1,50 @@
-import type { IOutboundProvider } from '@rocket.chat/core-typings';
-import { ajv } from '@rocket.chat/rest-typings';
-
 import { API } from '../../../../../app/api/server';
-import { isGETOutboundProviderParams } from '../outboundcomms/rest';
+import {
+	GETOutboundProvidersResponseSchema,
+	isGETOutboundProviderParams,
+	IsOutboundProviderBadRequestErrorSchema,
+	isOutboundProviderMetadataSchema,
+} from '../outboundcomms/rest';
 import { outboundMessageProvider } from './lib/outbound';
 import type { ExtractRoutesFromAPI } from '../../../../../app/api/server/ApiClass';
 
-const outboundCommsEndpoints = API.v1.get(
-	'omnichannel/outbound/providers',
-	{
-		response: {
-			200: ajv.compile<{ providers: IOutboundProvider[] }>({
-				type: 'object',
-				properties: {
-					providers: {
-						type: 'array',
-						items: {
-							type: 'object',
-							properties: {
-								providerId: {
-									type: 'string',
-								},
-								providerName: {
-									type: 'string',
-								},
-								supportsTemplates: {
-									type: 'boolean',
-								},
-								providerType: {
-									type: 'string',
-								},
-								documentationUrl: {
-									type: 'string',
-								},
-							},
-						},
-					},
-				},
-			}),
+const outboundCommsEndpoints = API.v1
+	.get(
+		'omnichannel/outbound/providers',
+		{
+			response: {
+				200: GETOutboundProvidersResponseSchema,
+				400: IsOutboundProviderBadRequestErrorSchema,
```

Comment:
nitpick, feel free to ignore if not valid: why the naming of these 2 are different? some are upper case, some are lowercase, one starts with GET and the rest with Is or is 🤔

Reply:
Yeah, I'm actually unsure on how to name these Error responses...

The GET prefix for the endpoint request params feels right to me.

But the error responses... do you have any suggestions?

Reply:
what about
200: GETxxxxxxResponse
400: GETxxxxxErrorResponse